docs/source/updating_capabilities.md

```diff
@@ -0,0 +1,341 @@
+# Updating the capability level of a channel
+
+*Audience: network administrators, node administrators*
+
+If you're not familiar with capabilities, check out [Capabilities](./capabilities_concept.html)
+before proceeding, paying particular attention to the fact that **node versions
+must be upgrading to the required level for all affected nodes before enabling
+capabilities**.
+
+For information about any new capability levels in the latest release of Fabric,
+check out [Upgrading your components](./upgrading_your_components.html#Capabilities).
```

Comment:
#Capabilities link doesn't exist.... also that topic doesn't have the per release capability updates mentioned, does it? Did you intend to link to "upgrade_to_newest_version"?

Reply:
Yeah this was meant to be the upgrade to the newest version doc. Will fix with next push (after we resolve the upgrade dbs thing).